lib/tern.js

```diff
@@ -610,7 +610,7 @@
     function gather(prop, obj, depth, addInfo) {
       // 'hasOwnProperty' and such are usually just noise, leave them
       // out when no prefix is provided.
-      if (query.omitObjectPrototype !== false && obj == srv.cx.protos.Object && !word) return;
+      if ((ignoreObj || query.omitObjectPrototype !== false) && obj == srv.cx.protos.Object && !word) return;
       if (query.filter !== false && word &&
```

Comment:
Why `ignoreObj` and not `objLit`? The latter seems to more precisely describe what we are talking about (and it looks like they'll have the same value at this point).